src/java.desktop/windows/native/libawt/windows/awt_Window.cpp

```diff
@@ -1419,17 +1422,19 @@ BOOL AwtWindow::UpdateInsets(jobject insets)
             LONG style = GetStyle();
             if (style & WS_THICKFRAME) {
                 m_insets.left = m_insets.right =
-                    ::GetSystemMetrics(SM_CXSIZEFRAME);
+                    ::GetSystemMetrics(SM_CXSIZEFRAME) +
+                     extraPaddedBorderInsets;
```

Comment:
If you put them on the same line, it nearly fits 80-column limit. Does it make sense to do so?